src/siembol/templates/deployment-rest.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ include "siembol.rest.fullname" $ }}
+  name: {{ include "siembol.rest.fullname" $ }}
+spec:
+  replicas: 1 
+  selector:
+    matchLabels:
+      app: {{ include "siembol.rest.fullname" $ }}
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: {{ include "siembol.rest.fullname" $}}
+      name: {{ include "siembol.rest.fullname" $ }}
+    spec:
+      containers:
+      - image: {{ .Values.rest.image.repository }}:{{ .Values.rest.image.tag }}
+        workingDir: {{ .Values.rest.mountPath }}
+        imagePullPolicy: {{ default "IfNotPresent" .Values.rest.imagePullPolicy | quote }}
+        name: {{ include "siembol.rest.fullname" $ }}
+        ports:
+          - containerPort: 8081
+        resources:
+          requests:
+            memory: "256Mi"
+            cpu: "500m"
+          limits:
+            memory: "1024Mi"
+            cpu: "1000m"
+        securityContext:
+          runAsUser: 101
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 8081
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 30
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 8081
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 10
+        env:
+        {{ if .Values.tls.enabled -}}
+        - name: JAVA_OPTS
+          value: -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts 
```

Comment:
is it needed?

Reply:
Which part, the `GIT_PSWD` or the `cacerts` `JAVA_OPTS`?  I think both are, no? 

We set the `GIT_PSWD` from the secret that we create for the cluster.  That secret holds the GitHub PAT.

The cacerts part needs to be there if TLS is enabled.  

Am I missing something?

Reply:
This was just comment to use ssl certs store in config editor rest. I think we do not need since config editor rest is not communicating with zk, siembol response using tls in this deployment